src/app/feedback/feedback.component.html

```diff
@@ -0,0 +1,31 @@
+<h1 i18n>Add feedback</h1>
+<form [formGroup]="feedbackForm" (ngSubmit)="addfeedback(feedbackForm.value)">
+  <div>
+    <label i18n>Is your feedback Urgent?</label>
+    <mat-radio-group formControlName="isUrgent">
+      <mat-radio-button value="true" i18n>Yes</mat-radio-button>
+      <mat-radio-button value="false" i18n>no</mat-radio-button>
+    </mat-radio-group>
+  </div>
+  <div>
+    <label i18n>Feedback Type:</label>
+    <mat-radio-group formControlName="feedbackType">
+      <mat-radio-button value="Question" i18n>Question</mat-radio-button>
+      <mat-radio-button value="Bug" i18n>Bug</mat-radio-button>
+      <mat-radio-button value="Suggestion" i18n>Suggestion</mat-radio-button>
+    </mat-radio-group>
+  </div>
+  <div>
+    <label i18n>Comment:</label>
+    <mat-form-field>
+      <textarea matInput placeholder="Your Feedback" formControlName="feedbackMsg" required></textarea>
+      <mat-error><planet-form-error-messages [control]="feedbackForm.controls.feedbackMsg"></planet-form-error-messages></mat-error>
+    </mat-form-field>
+  </div>
+  <div *ngIf="message"[ngClass]="(fedksuccess===true)?'alert alert-success':'alert-danger'" i18n>
```

Comment:
The `ngClass` is adding Bootstrap classes and should be removed.

Removing this will remove the two below issues, but I wanted to make a note of them so you can be careful to not make these errors again:
* There is a missing space between the `ngIf` and `ngClass`.
* The `fedksuccess` variable is redundant.  The `message` variable is storing whether or not the submission was a success, so `message === 'feedbackSuccess'` is equivalent and simplifies your component.